### PR TITLE
Revert Focus pattern inserter search when activating zoom out inserter

### PR DESCRIPTION
**Pull request by getdave**
## What?

Revert https://github.com/WordPress/gutenberg/pull/64396

## Why?

This added a React `Ref` in Redux state which is [an anti-pattern](https://redux.js.org/faq/organizing-state#can-i-put-functions-promises-or-other-non-serializable-items-in-my-store-state).

This was throwing an error like `t.toString`.

## How?

Revert to remove the ref in Redux state.

## Testing Instructions

- Open the global inserter 
- Click about in the canvas
- Toggle the tabs in the Inserter
- Basically ensure that `t.toString` doesn't not happen.

### CODE REVIEW

**Review by scruffian — APPROVED** [review]
Thanks for this. LGTM
